Makefile

```diff
@@ -220,6 +220,163 @@ ubuntu_docker_build: Dockerfile-ubuntu.$(PLATFORM).tag
+		$(CUSTOM_ARGS) \
+		$(CENTOS_DOCKER_IMAGE_NAME) \
+		-c ". activate datatable-py36-with-pandas && \
+			python --version && \
+			mkdir -p dist && \
+			make version > dist/VERSION.txt && \
+			cat dist/VERSION.txt"
+
+centos7_test_in_docker_impl:
+	docker run \
+		--rm \
+		--init \
+		-u `id -u`:`id -g` \
+		-v `pwd`:/dot \
+		-w /dot \
+		--entrypoint /bin/bash \
+		-e "DT_LARGE_TESTS_ROOT=$(DT_LARGE_TESTS_ROOT)" \
+		$(CUSTOM_ARGS) \
+		$(CENTOS_DOCKER_IMAGE_NAME) \
+		-c ". activate $(TEST_VENV) && \
+			export LD_LIBRARY_PATH=$$LD_LIBRARY_PATH:/opt/h2oai/dai/python/lib/ && \
+			python --version && \
+			pip install --no-cache-dir --upgrade dist/*.whl && \
+			make MODULE=datatable test_install && \
+			make test"
+
+centos7_test_py36_with_pandas_in_docker:
+	$(MAKE) TEST_VENV=datatable-py36-with-pandas centos7_test_in_docker_impl
+
+centos7_test_py35_with_pandas_in_docker:
+	$(MAKE) TEST_VENV=datatable-py35-with-pandas centos7_test_in_docker_impl
+
+centos7_test_py36_with_numpy_in_docker:
+	$(MAKE) TEST_VENV=datatable-py36-with-numpy centos7_test_in_docker_impl
+
+centos7_test_py36_in_docker:
+	$(MAKE) TEST_VENV=datatable-py36 centos7_test_in_docker_impl
+
+ubuntu_build_in_docker_impl:
+	docker run \
+		--rm \
+		--init \
+		-u `id -u`:`id -g` \
+		-v `pwd`:/dot \
+		-w /dot \
+		--entrypoint /bin/bash \
+		-e "CI_VERSION_SUFFIX=$(CI_VERSION_SUFFIX)" \
+		-e "DT_LARGE_TESTS_ROOT=$(DT_LARGE_TESTS_ROOT)" \
+		$(CUSTOM_ARGS) \
+		$(UBUNTU_DOCKER_IMAGE_NAME) \
+		-c ". /envs/$(BUILD_VENV)/bin/activate && \
+			python --version && \
+			make dist"
+
+ubuntu_build_sdist_in_docker:
+	docker run \
+		--rm \
+		--init \
+		-u `id -u`:`id -g` \
+		-v `pwd`:/dot \
+		-w /dot \
+		--entrypoint /bin/bash \
+		-e "CI_VERSION_SUFFIX=$(CI_VERSION_SUFFIX)" \
+		-e "DT_LARGE_TESTS_ROOT=$(DT_LARGE_TESTS_ROOT)" \
+		$(CUSTOM_ARGS) \
+		$(UBUNTU_DOCKER_IMAGE_NAME) \
+		-c ". /envs/datatable-py36-with-pandas/bin/activate && \
+			python --version && \
+			make sdist"
+
+ubuntu_build_py36_in_docker:
+	$(MAKE) BUILD_VENV=datatable-py36-with-pandas ubuntu_build_in_docker_impl
+
+ubuntu_build_py35_in_docker:
+	$(MAKE) BUILD_VENV=datatable-py35-with-pandas ubuntu_build_in_docker_impl
+
+ubuntu_coverage_py36_with_pandas_in_docker:
+	docker run \
+		--rm \
+		--init \
+		-u `id -u`:`id -g` \
+		-v `pwd`:/dot \
+		-w /dot \
+		--entrypoint /bin/bash \
+		-e "DT_LARGE_TESTS_ROOT=$(DT_LARGE_TESTS_ROOT)" \
+		$(CUSTOM_ARGS) \
+		$(UBUNTU_DOCKER_IMAGE_NAME) \
+		-c ". /envs/datatable-py36-with-pandas/bin/activate && \
+			python --version && \
+			make coverage"
+
+ubuntu_test_in_docker_impl:
+	docker run \
+		--rm \
+		--init \
+		-u `id -u`:`id -g` \
+		-v `pwd`:/dot \
+		-w /dot \
+		--entrypoint /bin/bash \
+		-e "DT_LARGE_TESTS_ROOT=$(DT_LARGE_TESTS_ROOT)" \
+		$(CUSTOM_ARGS) \
+		$(UBUNTU_DOCKER_IMAGE_NAME) \
+		-c ". /envs/$(TEST_VENV)/bin/activate && \
+			python --version && \
+			pip freeze && \
+			pip install --no-cache-dir --upgrade dist/*.whl && \
```

Comment:
Do we need to call --upgrade? In the container, they should not be any datatable.whl installed.
Furthermore, it is better to reference directly by name instead of wildcard (if it is possible).